### DISCUSSION TITLE
Admin Settings Page

### DISCUSSION THREAD

**Discussion by joshuadavidnelson** [joshuadavidnelson/disable-blog#44] [Ideas]
We're looking to merge Disable Blogging into Disable Blog (see the [Project](https://github.com/joshuadavidnelson/disable-blog/projects/3)).

A key feature that appears both on Disable Blog's to-do list (#39) and as an existing user interface on Disable Blogging is a settings page. So we're on the hunt for an API that we can use to build the new admin page with the mindset of using the many filters in Disable Blog to toggle features on/off similar to Disable Blogging. Opening this up as our first Discussion, if there are any recommendations, questions, etc.

**Comment by joshuadavidnelson** [comment]
@ethanosullivan I'm currently playing around with this one: https://github.com/jamesckemp/WordPress-Settings-Framework

**Reply by jinksraevan** [reply]
I just checked out the repo. It looks like it's still maintained and easy enough to implement. Let's try this one out with some basic options to test.

**Reply by joshuadavidnelson** [reply]
I'm working on a local branch for this now, I'll ping you when I have something to show for it 😄
